CalcualtorLogic.js

```diff
@@ -119,8 +119,7 @@ class Calculator {
     onOperatorButtonClick(event) {
         this.operator = event.target.dataset.operator;
         this.lastButtonClicked = "operator";
-        this.lastValue = this.currentResult + this.operator;
-        console.log(this.newValue)
+        this.lastValue = this.currentResult;
```

Comment:
This is correct 
This what I was expecting to do from the beginning 
1. save the clicked operator in `this.operator`
2. set flag indicates that the last button is clicked was an "operator" 
3. copy the current result to another property 

Now that you implemented this correctly.
I want you to add a comment for each line exlain what you did and why
later when you come back to this you can read those comment to understand your code.